BackofficeBundle/Tests/EventSubscriber/UpdateRouteDocumentSubscriberTest.php

```diff
@@ -129,4 +129,22 @@ public function testUpdateRouteDocumentOnSiteUpdate()
         Phake::verify($this->objectManager)->remove($route);
         Phake::verify($this->objectManager)->flush();
     }
+
+    /**
+     * Test on deleteForRedirection
+     */
+    public function testDeleteForRedirection()
+    {
+        $redirection = Phake::mock('OpenOrchestra\ModelInterface\Model\RedirectionInterface');
+        $event = Phake::mock('OpenOrchestra\ModelInterface\Event\RedirectionEvent');
+        $route = Phake::mock('OpenOrchestra\ModelInterface\Model\RouteDocumentInterface');
+
+        Phake::when($event)->getRedirection()->thenReturn($redirection);
+        Phake::when($this->routeDocumentManager)->deleteForRedirection(Phake::anyParameters())->thenReturn(array($route));
```

Comment:
You could return `array($route, $route)` instead to be sure that the foreach loop is used